R/ae_table_soc.R

```diff
@@ -65,18 +65,19 @@ ae_table_soc = function(
     df_ae, ..., df_enrol,
     variant=c("max", "sup", "eq"),
     arm=NULL, term=NULL,
+    ae_groups = NULL,
+    cols_to_show = NULL,
```

Comment:
Est-ce qu'on a besoin d'avoir un argument `cols_to_show` en plus ?
Pour moi si `!is.null(ae_groups)`, on ne montre que les colonnes dans `names(ae_groups)`.

Je n'ai jamais vu de tableau qui gardait toutes les colonnes, mais en vrai c'est pas mal (quand il y a un seul bras).
Quand même, on pourrait l'écrire comme ça, non ?
```r
ae_groups = list(
  "G1"=1, "G2"=2, "G3"=3, "G4"=4, "G5"=5,
  "Grade [1-2]" = 1:2,
  "Grade [1-3]" = 1:3,
  "Grade [3-5]" = 3:5
)
```